blue-green-deploy.sh

```diff
@@ -0,0 +1,52 @@
+"""The blue-green deploy plug-in that works with build packs is deprecated,
+So this script reproduces the steps in a shell script. From: https://github.com/concourse/cf-resource/issues/62
```

Comment:
Tried out this script with a push to prod (from my command line, not CircleCI) – worked great! 

Love how this gets us zero downtime, I could see the difference between this strategy and the current strategy which leaves us with a period of `404 Not Found: Requested route does not exist` during deploys. 

😍